slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/SignedAttestationsDao.java

```diff
@@ -153,4 +153,20 @@ public void insertAttestation(final Handle handle, final SignedAttestation signe
         .mapTo(UInt64.class)
         .findFirst();
   }
+
+  public Optional<UInt64> maximumTargetEpoch(final Handle handle, final int validatorId) {
+    return handle
+        .createQuery("SELECT MAX(target_epoch) FROM signed_attestations WHERE validator_id = ?")
```

Comment:
Can you add some Dao tests for this and the maximumSourceEpoch functions

Reply:
removed